test/protractor/pages/settings.js

```diff
@@ -0,0 +1,119 @@
+var bedrock = global.bedrock;
+
+var api = {};
+module.exports = api;
+
+var by = global.by;
+var element = global.element;
+var should = global.should;
+var expect = global.expect;
+var protractor = global.protractor;
+
+api.PRIVACY_NO_PUBLIC = 0;
+api.PRIVACY_SPECIFIED_PUBLIC = 1;
+api.PRIVACY_PUBLIC_EXCEPT_EMAIL = 2;
+api.PRIVACY_ALL_PUBLIC = 3;
+
+api.LABEL_MODEL = '$ctrl.identityChanges.label';
+api.EMAIL_MODEL = '$ctrl.identityChanges.email';
+api.WEBSITE_MODEL = '$ctrl.identityChanges.url';
+api.IMAGE_TYPE_MODEL = '$ctrl.identityChanges.sysImageType';
+api.IMAGE_URL_MODEL = '$ctrl.identityChanges.image';
+api.GRAVATAR_MODEL;
+api.DESCRIPTION_MODEL = '$ctrl.identityChanges.description';
+api.PRIVACY_MODEL = '$ctrl.publicMode';
+
+api.get = function(slug) {
+  bedrock.get('/i/' + slug);
+  return api;
+};
+
+api.label = function(label) {
```

Comment:
Seems like this should be named `label`, but we could make these sorts of things double as getters and setters in the future or elsewhere so I think I'm ok with it as is.